.vnurc

```diff
@@ -4,7 +4,7 @@
 # Proposed, tracking in gh-429
 Bad value “” for attribute “aria-activedescendant” on element “ul”:.*
```

Comment:
issue 429 is closed, and the related validator issue is closed too. Can we remove this?